src/otx/algorithms/detection/adapters/mmdet/apis/train.py

```diff
@@ -119,11 +121,26 @@ def train_detector(model, dataset, cfg, distributed=False, validate=False, times
     elif cfg.device == "xpu":
         model = build_dp(model, cfg.device, device_ids=cfg.gpu_ids, enable_autocast=bool(fp16_cfg))
         model.to(f"xpu:{cfg.gpu_ids[0]}")
+    elif cfg.device == "hpu":
+        import habana_frameworks.torch.core as htcore
+        from habana_frameworks.torch.utils.library_loader import load_habana_module
+
+        load_habana_module()
+        os.environ["PT_HPU_LAZY_MODE"] = "1"
+        assert len(cfg.gpu_ids) == 1
+        model = build_dp(
+            model, cfg.device, device_ids=cfg.gpu_ids, dim=0, enable_autocast=bool(fp16_cfg), put_gt_on_device=False
+        )
+        model.to(model.src_device_obj)
+        htcore.mark_step()
+        model.zero_grad()
```

Comment:
Could you explain why they are here? AFAIK, they can be in train/eval loop.

Reply:
I just followed some code examples in Habana documentation. I don't think we need it either. I will remove this